examples/gno.land/p/demo/json/node_test.gno

```diff
@@ -285,7 +285,7 @@ func TestNode_GetBool(t *testing.T) {
 	}
 }
 
-func TestNode_GetBool_Fail(t *testing.T) {
```

Comment:
i was personnally fine looking for `Fail` which is different from `FAIL`, but ok :)